src/cat.js

```diff
@@ -32,12 +32,9 @@ function _cat(options, files) {
     if (!fs.existsSync(file))
       common.error('no such file or directory: ' + file);
 
-    cat += fs.readFileSync(file, 'utf8') + '\n';
+    cat += fs.readFileSync(file, 'utf8');
   });
 
-  if (cat[cat.length-1] === '\n')
```

Comment:
@ariporad I'm not sure what you intend to be the correct behavior. I tried your example, and I see a newline above the prompt.

The line I removed would strip a trailing newline that the old code incorrectly appended. The code doesn't append newlines anymore, so it doesn't need to strip a trailing newline.